taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -358,7 +410,18 @@ def deliver_project(self):
                 except Exception as e:
                     logger.warning(
                         "failed to create final aggregate report for {}, "\
-                        "reason: {}".format(self,e))
+                        "reason: {}".format(self, e))
```

Comment:
The attribute error is raised if `report_aggregate`is not present in the config, which is then interpreted as reports not being desired, i.e. they are opt-in to generate